dev/devicelab/lib/framework/manifest.dart

```diff
@@ -118,7 +118,7 @@ ManifestTask _validateAndParseTask(dynamic taskName, dynamic taskYaml) {
   );
 }
 
-List<String> _validateAndParseCapabilities(String taskName, dynamic capabilitiesYaml) {
+List<dynamic> _validateAndParseCapabilities(String taskName, dynamic capabilitiesYaml) {
```

Comment:
Since this function performs the type checks, we should be able to cast it to `List<String>` using the new [List.cast method](https://docs.flutter.io/flutter/dart-core/List/cast.html). I think we should be able to do the same with the `Map` below

Reply:
Added the cast method for the list.

Not sure I understand the comment about the Map below as the _checkKeys method validates the types but does not return any  object.

Reply:
> Not sure I understand the comment about the Map below as the _checkKeys method validates the types but does not return any object.

Oh, you're right. Nothing to do in the `Map` case.